### PR TITLE
feat: people for identity in paseo system chains

### PR DESCRIPTION
**Pull request by al3mart**
Changes to use people chain as the source for identities in Paseo.
Also includes some new RPCs.

### CODE REVIEW

**Comment by al3mart** [comment]
This is ready for review @TarikGul

File: packages/apps-config/src/endpoints/testingRelayPaseo.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export const testParasPaseo: Omit<EndpointOption, 'teleport'>[] = [
   {
     homepage: 'https://ajuna.io/',
     info: 'Ajuna(paseo)',
+    isPeopleForIdentity: true,
```
[the remark this answers is not in the document]
Comment (al3mart) [reply packages/apps-config/src/endpoints/testingRelayPaseo.ts:37]: Thanks, wasn't aware :+1:

[the remark this answers is not in the document]
Comment (al3mart) [reply packages/apps-config/src/endpoints/testingRelayPaseo.ts]: My last commit addresses this.

**Review by TarikGul — APPROVED** [review]
LGTM, thanks so much!

**Pull request merged**